builds/android/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -1213,10 +1221,10 @@ else if (event.getAction() == KeyEvent.ACTION_UP) {
     }
 
     // Touch events
-    /* EASYRPG: TOUCH DISABLED
+    /* EasyRPG modification: disable touch input
     @Override
     public boolean onTouch(View v, MotionEvent event) {
-        // Ref: http://developer.android.com/training/gestures/multi.html
+        /* Ref: http://developer.android.com/training/gestures/multi.html */
```

Comment:
I think the strange compile error is this. That nice /* */ one-line-comment destroys your multi-line comment